app/models/katello/authorization/subscription.rb

```diff
@@ -4,13 +4,11 @@ module Authorization::Subscription
 
     include Authorizable
 
-    def readable?
```

Comment:
The tests are passing => this code my be dead (or untested). that said, I'd rather leave it in since it's not related to this change.

Reply:
@jturel ok, adding it back